Cargo.toml

```diff
@@ -17,6 +17,7 @@ members = [
 wsts = "7.0"
 rand_core = "0.6"
 rand = "0.8"
+sha2 = "0.10"
```

Comment:
I think you need to change this to 
```
[target.'cfg(all(any(target_arch = "x86_64", target_arch = "x86", target_arch = "aarch64"), not(target_env = "msvc")))'.dependencies]
sha2 = { version = "0.10", features = ["asm"] }

[target.'cfg(any(not(any(target_arch = "x86_64", target_arch = "x86", target_arch = "aarch64")), target_env = "msvc"))'.dependencies]
sha2 = { version = "0.10" }
```

And then update all other instances in stacks core where the cargo tomls reference sha2, to reference the workspace one instead

Reply:
I tried this but got an error. Think the target specification can only be placed under individual crates. If that's the case, I will remove sha2 from here and add the target one to stacker-signer's Cargo.toml

Reply:
ah that might be the case. i have never tried using workspace when accessing target specific shas...yes you can ignore my intial comment and just use it in the stacks-signer sub dir as you had originally :)

Reply:
I don't think you need this dependency at all.  You should be able to use one of the hash functions in `stacks-common/src/util/hash.rs`.